README.md

```diff
@@ -520,51 +520,76 @@ The following command line options are available
   -s    
   -skipVersionCheck
         Indicates the service should skip the Core Service's version compatibility check.
+  -sk
+  --serviceKey                
+        Overrides the service key to be used with Registry and/or Configuration Provider.
+        If the name provided contains the text `<profile>`, this text will be replaced
+        with the name of the profile used.
 ```
 
 Examples:
```

Comment:
Should we add an example of flag `-sk` below?

Reply:
I didn't since we don't have examples for any of the command-line options and feel the text explains it well enough.